libbeat/otelbeat/oteltranslate/tls_otel_test.go

```diff
@@ -0,0 +1,229 @@
+aWVudDEgMB4GCSqGSIb3DQEJARYRY2xpZW50QGVsYXN0aWMuY28wggEiMA0GCSqG
+SIb3DQEBAQUAA4IBDwAwggEKAoIBAQDAVGPlx4U2BpWfQlyMNraLMjdJAo4PjO2G
+rDbwg2cAO4QFbMECEiNHakvuJ3zVVDO+HsBdkLWr8nO4iXmZDokfDrOrANJuqq16
+p022soC8pJQz9uIBWTnxDGd/wdofi4H+V5uaMhw961sgB7GREyBWNRBQzhcFyQEP
+XkR1/G52PcuzM5H9cnOSy7jc62g8Pkk8c2eZu3ADmvgWSH0b5pFUIvKsq068QjKP
+qoHYXn38d/SSeCX57tzKsj+mBzp0cr1f9jeXmKeu68wPYG14aj9WmmY6ICAPvqPF
+BKNLhXn2xPlZzv93zjiUR5bnitenVxvsmwjn5XvlgH56/fh3Y3aPAgMBAAGjLDAq
+MAkGA1UdEwQCMAAwHQYDVR0OBBYEFJlsmqi3qid9YoWj4N7GQvAywRzbMA0GCSqG
+SIb3DQEBCwUAA4IBAQCAtBwyiRYAGeAcN/UuMEcnMXP8QNrnCO/unoCbyFsByFQT
+TcwMrS441hGPp/cAa8Fx0cP+oqrO99G1YHCzhprYVqIi/W9MsvRnR7Nh8SSS2/ld
+0Gv9g+DU89NMzE5hlMCt5V0ydKbRj+ChKDsKlgQSopbrArjxHQv4Hb234HSZAR5N
+OkJ1rNCF7wMD+xlNzEWZAHl7qjHuG8C4xWP207dXGYuY3064rBqv9hypLxj7RuZn
+qesVBabxXBCL6Y1foh5OLLHyEWw28yfK/PnVdqU0lLrBhW9VJ6mQ9XCwZxf/tlSk
+B3FafTQk4ZtU+4bVJuiAiQI7DeqpIFU6Lczds2gG
+-----END CERTIFICATE-----
+`, // the following key pem is decrypted value using key_passphrse
+				"key_pem": `-----BEGIN RSA PRIVATE KEY-----
+MIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQDAVGPlx4U2BpWf
+QlyMNraLMjdJAo4PjO2GrDbwg2cAO4QFbMECEiNHakvuJ3zVVDO+HsBdkLWr8nO4
+iXmZDokfDrOrANJuqq16p022soC8pJQz9uIBWTnxDGd/wdofi4H+V5uaMhw961sg
+B7GREyBWNRBQzhcFyQEPXkR1/G52PcuzM5H9cnOSy7jc62g8Pkk8c2eZu3ADmvgW
+SH0b5pFUIvKsq068QjKPqoHYXn38d/SSeCX57tzKsj+mBzp0cr1f9jeXmKeu68wP
+YG14aj9WmmY6ICAPvqPFBKNLhXn2xPlZzv93zjiUR5bnitenVxvsmwjn5XvlgH56
+/fh3Y3aPAgMBAAECggEAKbvmNX03BcMmAnn29SIGOGw8HOamBu/QtvF1tnj9B8ri
+Wf8AXr+q0htZwKLm7q+nzrCDk4oMMfSZcci7DyBdVtTs3cV+5C67GCtnrKZNUyHv
+WttOrXY8IXdMmidpeoDeQ1+lTy9ie3kvu+KPgGiDEtHO6Yne6w1z4m7VMjkFizh6
+Y5sVPGTV1anoiO5tpW/D/Pshh/u+ZVsaeIE0iKV2NACJoMC0opsXBslldBDvXIaV
+ZhpEnr5v3CO8i3roq2qp4Bplfyy8g89KXerg6G4JxM0NSafuWwGC/13chHSw3ke1
+5odqnnwUyJ+MNvzyH0CPzF56vE3hFe6byeG3rD2euQKBgQD+MPtpvYGZfvMdVKNu
+zms4s9CLcoM6lJXD72xL73x4MUcPQLOfk2YpCLDuVadCA8jBvL7d0OjulYbjJNCX
+XZH2CITtTPzDEn+hfxKcy4Jd2M53yMQvpr55nmysEp+juuhkAo/xlXgGOcL3nq7o
+hBLEAMy2NKwkszt1VLyZaD9SPQKBgQDBsrmb9INYJsCboA/VydKtzMB7Nnrck4S6
+1BOsEp5JkepHPC92yolPCI94hJlTdISbt69nbcCGDvD87j2Z2UVP4zOfxT/SGHmF
+LgLXgqzVCYcgvpBXZey9rPWvEPwQG/0ODgva7GxHnmVbP0LrQ4ynnX6FMsWaBYFd
+K/0eEwY0uwKBgQCieGpqBr+sfbEk4TFpJLTx1DUKvJHWQpyLVSAyVQuIw254+FEX
+QR5+QdjdLZAvqL2L33lbzCjmPlquGpzc8ujVilJ0Xs38XXmInvElmQpls6sccw26
+q2h50eICBhFVlKTvL5gTwQarbAYLQbjoU2qvLxepqncRKiJp91Ro9XHrvQKBgG9K
+mFiiEcFZartAKTkF4BXaGhHxSIBqBg4ugisQ+3975icNzpurXV9apMxzK4GG5hZu
+YMrFhaPA+/fnjt9RtgBjo6q986BsTY4W1K0suM8izVAkDd0Zg/+rW/I9iQZcfnZP
+3cHoq4Iu4T+fRnzUcAFyfVpcxKptVVnKR4G7Hoq3AoGBAPrrQtt4QvmAxPknKFgO
+mDke0HqyWowuMkAHoVLL3VUaH8hrI1FA+oWEXpor4VqguB8D1SWb0qAJRF7pIxoE
+m8TtceUhSOnXNrrO5agyMRmL0aYf8D425ot/uwTiSkOd4bdFeEaYs0ahHosxHq2N
+me1zqwZ6EX7XHaa6j1mx9tcX
+-----END RSA PRIVATE KEY-----
+`,
+				"cipher_suites":                []string{},
+				"insecure_skip_verify":         false,
+				"include_system_ca_certs_pool": false,
+			},
+			err: false,
+			// TODO: Add  more  scenarios
```

Comment:
Is this supposed to be addressed before merging?

Reply:
no, this is also for future reference to come back to it and add more scenarios